doc/onboarding.md

```diff
@@ -3,6 +3,12 @@
 This document is an outline of the things we tell new Collaborators at their
 onboarding session.
 
+## One week before the onboarding session
+
+* Ask the new Collaborator if they are using two-factor authentication on their GitHub account. If they are not, suggest that they enable it as their account will have elevated privileges in mamy of the Node.js repositories.
```

Comment:
@jbergstroem Let's keep it as «suggest» for now and re-evaluate later, there could be some issues with enabling 2FA that we are not aware of, and we don't want to scare the new people from being added as collaborators.

Once we collect some feedback/results on this process, we could make it stricter, either by «strongly suggest» or by even enforcing a hard requirement.